build/yaml/cleanupResources/cleanupResources.yml

```diff
@@ -18,6 +18,7 @@ variables:
   InternalAppServicePlanDotNetName: "bffnbotsappservicedotnet$($env:RESOURCESUFFIX)"
   InternalAppServicePlanJSName: "bffnbotsappservicejs$($env:RESOURCESUFFIX)"
   InternalAppServicePlanPythonName: "bffnbotsappservicepython$($env:RESOURCESUFFIX)"
+  InternalVirtualNetworkName: "bffnvirtualnetwork$($env:RESOURCESUFFIX)"
```

Comment:
We should put this variable after the `InternalSharedResourceGroupName` because they are ordered alphanumerically